pkg/connman/connman.go

```diff
@@ -27,7 +28,12 @@ func New() *ConnectionManager {
 func (m *ConnectionManager) Set(key string, conn net.Conn) {
 	dialer := revdial.NewDialer(conn, "/ssh/revdial")
 
-	m.dialers.Store(key, dialer)
+	if size := m.dialers.Store(key, dialer); size > 1 {
+		logrus.WithFields(logrus.Fields{
+			"key":  key,
+			"size": size,
+		}).Warning("Multiple connections stored for the same identifier.")
```

Comment:
Other important fields to add to the log?

Reply:
In this context, there are no additional fields that can be added to the log